scripts/update-aws-native.sh

```diff
@@ -2,27 +2,26 @@
 
 set -euo pipefail
 
-# Require that VERSION environment variable is set
-if [ -z "${VERSION}" ]; then
-  echo "VERSION environment variable is required to set the desired pulumi-aws-native version (no v prefix)"
-  exit 1
-fi
-
 # Update references in examples/
 for e in $(find examples -name package.json | grep -v node_modules);
 do
     echo "Updating $e"
-    (cd $(dirname $e) && yarn install)
-    (cd $(dirname $e) && yarn upgrade "@pulumi/aws-native@^${VERSION}")
+    dir=$(dirname $e)
+    npx ncu --filter @pulumi/aws-native --upgrade --cwd $dir
 done
 
 for e in $(find integration -name package.json | grep -v node_modules);
 do
     echo "Updating $e"
-    (cd $(dirname $e) && yarn install)
-    (cd $(dirname $e) && yarn upgrade "@pulumi/aws-native@^${VERSION}")
+    dir=$(dirname $e)
+    npx ncu --filter @pulumi/aws-native --upgrade --cwd $dir
```

Comment:
Switching to [ncu](https://www.npmjs.com/package/npm-check-updates) since it handles upgrading to the latest minor version.